bukkit/src/main/java/com/viaversion/viaversion/bukkit/providers/BukkitPickItemProvider.java

```diff
@@ -187,14 +189,15 @@ static BlockToItem build() {
                     }
                     return item;
                 };
-            } else if (PaperViaInjector.hasMethod(Material.class, "isItem")) {
-                return (block, includeData) -> new ItemStack(block.getType(), 1);
-            } else {
+            } else if (Via.getAPI().getServerVersion().lowestSupportedProtocolVersion().equalTo(ProtocolVersion.v1_8)) {
                 LegacyBlockToItem legacy = LegacyBlockToItem.getInstance();
                 return legacy != null ? (block, includeData) -> legacy.blockToItem(block) : (b, i) -> null;
+            } else if (PaperViaInjector.hasMethod(Material.class, "isItem")) {
+                return (block, includeData) -> block.getType().isItem() ? new ItemStack(block.getType(), 1, (short) 0, block.getData()) : null;
```

Comment:
On 'normal' Spigot/Paper servers, this was added *after* block data was already removed. With the edited version check above, this should just use the type without data anyways

Reply:
1.20 
![image](https://github.com/user-attachments/assets/1bf5747d-2661-4113-9c3a-34de9ca437a6)

https://helpch.at/docs/1.20/org/bukkit/block/Block.html

Reply:
Yes, it shouldn't be used after 1.12 / doesn't have to given the other version change